Development/xASL_test_BIDSFlavorsFull.m

```diff
@@ -126,6 +140,9 @@ function xASL_test_BIDSFlavorsFull(pathExploreASL,pathTest,bTest)
 				end
 			else
 				warning('Here we expect a unix-ish system');
+                diary('off');
+                fclose('all'); % ensure that no file is locked
+                xASL_delete(fullfile(ListFolders{iList}, 'derivatives'));
```

Comment:
@jan-petr: I think I forgot to add **true** for the second argument here, so that it deletes the folder content as well. Can we fix this?